src/components/DayPickerNavigation.jsx

```diff
@@ -74,39 +89,49 @@ export default function DayPickerNavigation(props) {
   });
   const prevClassNames = cx('DayPickerNavigation__prev', {
     'DayPickerNavigation__prev--default': isDefaultNavPrev,
+    'DayPickerNavigation__prev--rtl': isRTL,
   });
   const nextClassNames = cx('DayPickerNavigation__next', {
     'DayPickerNavigation__next--default': isDefaultNavNext,
+    'DayPickerNavigation__next--rtl': isRTL,
   });
 
+
+  const prevButton = () => (
```

Comment:
I think "verbose" isn't really that much of a problem; either these should just be inline, or they should be separate SFCs.